src/etc/di.xml

```diff
@@ -26,6 +26,10 @@
                 <item name="wishlist" xsi:type="string">ScandiPWA\Router\Validator\Wishlist</item>
                 <item name="checkout" xsi:type="string">ScandiPWA\Router\Validator\AlwaysPass</item>
                 <item name="compare" xsi:type="string">ScandiPWA\Router\Validator\AlwaysPass</item>
+                <item name="customer" xsi:type="string">ScandiPWA\Router\Validator\AlwaysPass</item>
+                <item name="sales" xsi:type="string">ScandiPWA\Router\Validator\AlwaysPass</item>
+                <item name="search" xsi:type="string">ScandiPWA\Router\Validator\AlwaysPass</item>
```

Comment:
do we want to always pass search? what if there's no results?

Reply:
Actually, it will be better to add this change, for consistency.
I'm not sure what is the desired behavior, though.

Earlier PWA would return 404 on PLP's on no products, and 200 if there were products.
Search behaves the same way now:
https://luma-demo.scandipwa.com/search/yoloqwe <--- 404
https://luma-demo.scandipwa.com/search/yoloqwe <-- 200

However, it looks like at some point it was changed, and now PLP's always return 200.
Then we may as well do the same for search.
However, if we do that (and it is a desired result), IMO would need to remove the 200/404 status check logic that relates to product count, as that's extra logic that wouldn't be used.

Overall -> I'd consult SEO team, what they want there, 200 or 404, and go with that on both PLP and search.